TM1py/Services/HierarchyService.py

```diff
@@ -636,7 +639,10 @@ def update_or_create_hierarchy_from_dataframe(
 
         if unwind:
             self.remove_all_edges(dimension_name, hierarchy_name)
-
+            
+        if unwind_consol.strip() != '' and unwind == False:
```

Comment:
Keeping `unwind` as **False** and providing a value for `unwind_consol` is misleading.
It would be more intuitive if we combine them into one argument or atleast keep the `unwind` as **True** and then based on `unwind_consol` value, perform full or partial unwind.

Reply:
> Keeping `unwind` as **False** and providing a value for `unwind_consol` is misleading. It would be more intuitive if we combine them into one argument or atleast keep the `unwind` as **True** and then based on `unwind_consol` value, perform full or partial unwind.

I have actually wanted to call _unwind_ as _unwind_all_ but this means any existing script using this function has to be modified. Agree that it is a bit confusing, but not sure if we can do anything with _unwind_ param at the moment. I might go for the second option you have suggested, i.e. Unwind = True and then Full or Partial unwind operation. Thanks for the tip!

Reply:
@rkvinoth you are suggesting this and then inside this, an iterator to unwind multiple rollups

        if unwind:
            if unwind_consol.strip() != '':
                self.remove_edges_under_consolidation(dimension_name, hierarchy_name,unwind_consol)
            else:
                self.remove_all_edges(dimension_name, hierarchy_name)